contracts/vault/CurveSwapper.sol

```diff
@@ -119,13 +96,15 @@ abstract contract CurveSwapper {
             swapper.underlyingDecimals
         );
 
-        return
-            swapper.pool.exchange_underlying(
-                swapper.tokenI,
-                swapper.underlyingI,
-                _amount,
-                minAmount
-            );
+        amount = swapper.pool.exchange_underlying(
```

Comment:
should there not be some check to make sure inputtoken actually is swappable to our underlying. ie part of some pool inserted into vault. if `swappers[_token]` is null. should we not revert with some message?

Reply:
I guess we can add a custom error message when the pool doesn't exist, yes